core/src/main/scala/org/apache/spark/deploy/history/HistoryServer.scala

```diff
@@ -179,7 +180,11 @@ class HistoryServer(
   }
```

Comment:
A note parallel to what @srowen is saying here, if you look at this change alongside your other pr #15247 `getApplicationInfoList` is actually no longer used. So either change how it works to avoid adding the new view function or deleting it completely would be good options